templates/email/default/waste/echo_failed_bulky_payment.txt

```diff
@@ -0,0 +1,40 @@
+Subject: Echo failed to confirm payment
+
+[%
+USE pounds = format('%.2f');
+PROCESS 'waste/_bulky_data.html';
+~%]
+
+Echo failed to confirm payment for the following bulky waste collection:
+
+[% report.detail %]
+
+Collection ref: [% report.id %]
+
+Collection date: [% collection_date %]
+
+Items to be collected:
+[% FOR item IN item_list %]
+    - [% item.value %]
+[% IF item.message %]
+        - [% item.message %]
+[% END %]
+[% END %]
+
+Amount paid: £[% pounds(payment / 100) %]
+
+Payment ref: [% report.get_extra_metadata('payment_reference') %]
+
+Payment type: [% report.get_extra_field_value('payment_method').replace( '_', ' ' ).ucfirst %]
+
+Customer details:
+
+    Name: [% report.name %]
+
+    Email: [% report.user.email %]
+
+    Phone: [% report.user.phone %] [% # TODO Where is this? %]
+
+Collection instructions: [% report.get_extra_field_value('Exact_Location') %]
+
+Error: [% # TODO Do we have access to any errors from Echo? %]
```

Comment:
No, if something has gone wrong, we'll only have an internal error I imagine.

Reply:
Should I be trying to fetch this internal error? Where would it be stored?

Reply:
No, it will literally be the phrase "Internal error" :)